Gemfile

```
@@ -1,22 +1,7 @@
 if ENV.fetch('BUNDLE_FOREMAN', '1') == '0'
   source 'https://rubygems.org'
-else
-  foreman_path = Dir['./foreman', '../foreman', '../../foreman']
-  foreman_path.map! { |path| File.join(path, 'Gemfile') }
-  foreman_gemfile = foreman_path.detect { |p| File.exist?(p) }
-  raise 'Foreman have not been found!' unless foreman_gemfile
 
-  foreman_gemfile = File.expand_path(foreman_gemfile)
-  eval_gemfile foreman_gemfile
+  gemspec name: 'foreman_puppet'
```

Usually it can detect this automatically
```suggestion
  gemspec
```